collect_app/src/main/java/org/odk/collect/android/receivers/NetworkReceiver.java

```diff
@@ -73,13 +74,23 @@ public void onReceive(Context context, Intent intent) {
             NetworkInfo ni = connectivityManager.getActiveNetworkInfo();
 
             if (ni != null && ni.isConnected()) {
-                uploadForms(context, isFormAutoSendOptionEnabled(currentNetworkInfo));
+                uploadForms(context, networkTypeMatchesAutoSendSetting(currentNetworkInfo));
             }
         }
     }
 
-    private boolean isFormAutoSendOptionEnabled(NetworkInfo currentNetworkInfo) {
-        // make sure autosend is enabled on the given connected interface
+    /**
+     * Returns whether the currently-available connection type is included in the app-level auto-send
+     * settings.
+     *
+     * @return true if a connection is available and settings specify it should trigger auto-send,
+     * false otherwise.
+     */
+    private boolean networkTypeMatchesAutoSendSetting(NetworkInfo currentNetworkInfo) {
```

Comment:
If I'm understanding this right, it seems a name like `isAutosendEnabledForNetworkType` would get across the idea better for me.  But let me know if I've misunderstood the intent.